product_docs/docs/epas/11/epas_rel_notes/index.mdx

```diff
@@ -23,28 +23,29 @@ EDB Postgres Advanced Server 11 is built on the open source PostgreSQL 11. EDB P
-| [11.18.29](epas11_18_29_rel_notes.mdx) | 10 Nov 2022 | [11.18](https://www.postgresql.org/docs/11/release-11-18.html)           |
-| [11.17.28](epas11_17_28_rel_notes.mdx) | 11 Aug 2022 | [11.17](https://www.postgresql.org/docs/11/release-11-17.html)           |
-| [11.16.26](epas11_16_26_rel_notes.mdx) | 12 May 2022 | [11.16](https://www.postgresql.org/docs/11/release-11-16.html)           | 
-| [11.15.25](09_epas11.15.25_rel_notes.mdx) | 10 Feb 2022 | [11.15](https://www.postgresql.org/docs/11/release-11-15.html)           | 
-| [11.14.24](10_epas11.14.24_rel_notes.mdx) | 11 Nov 2021| [11.14](https://www.postgresql.org/docs/11/release-11-14.html) | 
-| [11.13.23](11_epas11.13.23_rel_notes.mdx) | 08 Sep 2021| [11.13](https://www.postgresql.org/docs/11/release-11-13.html) | 
-| [11.12.22](13_epas11.12.22_rel_notes.mdx) | 05 May 2021| [11.12](https://www.postgresql.org/docs/11/release-11-12.html) | 
-| [11.12.21](15_epas11.12.21_rel_notes.mdx) | 15 Apr 2021| [11.12](https://www.postgresql.org/docs/11/release-11-12.html) | 
-| [11.11.20](17_epas11.11.20_rel_notes.mdx) | 12 Feb 2021| [11.11](https://www.postgresql.org/docs/11/release-11-11.html) | 
-| [11.10.19](19_epas11.10.19_rel_notes.mdx) | 20 Nov 2020| [11.10](https://www.postgresql.org/docs/11/release-11-10.html) | 
-| [11.9.17](21_epas11.9.17_rel_notes.mdx)   | 18 Aug 2020| [11.9](https://www.postgresql.org/docs/11/release-11-9.html)   | 
-| [11.9.16](23_epas11.9.16_rel_notes.mdx)   | 17 Aug 2020| [11.9](https://www.postgresql.org/docs/11/release-11-9.html)   | 
-| [11.8.15](25_epas11.8.15_rel_notes.mdx)   | 18 May 2020| [11.8](https://www.postgresql.org/docs/11/release-11-8.html)   | 
-| [11.7.14](27_epas11.7.14_rel_notes.mdx)   | 14 Feb 2020| [11.7](https://www.postgresql.org/docs/11/release-11-7.html) | 
-| [11.6.13](29_epas11.6.13_rel_notes.mdx)   | 19 Nov 2019| [11.6](https://www.postgresql.org/docs/11/release-11-6.html) | 
-| [11.5.12](31_epas11.5.12_rel_notes.mdx)   | 26 Aug 2019| [11.5](https://www.postgresql.org/docs/11/release-11-5.html) | 
-| [11.4.11](33_epas11.4.11_rel_notes.mdx)   | 25 Jun 2019| [11.4](https://www.postgresql.org/docs/11/release-11-4.html) | 
-| [11.3.10](35_epas11.3.10_rel_notes.mdx)   | 13 May 2019| [11.3](https://www.postgresql.org/docs/11/release-11-3.html) | 
-| [11.2.9](37_epas11.2.9_rel_notes.mdx)     | 22 Feb 2019| [11.2](https://www.postgresql.org/docs/11/release-11-2.html) | 
-| [11.1.7](39_epas11.1.7_rel_notes.mdx)     | 28 Nov 2018| [11.1](https://www.postgresql.org/docs/11/release-11-1.html) | 
\ No newline at end of file
+|                  Version                  | Release Date |                         Upstream Merge                         |
+| ----------------------------------------- | ------------ | -------------------------------------------------------------- |
+| [11.21.33](epas11_21_33_rel_notes.mdx)    | 25 Sep 2023  |                                                                |
```

Comment:
No upstream merge link?

Reply:
The release annoucement doesn't say there is any kind of upstream merge: https://docs.google.com/document/d/1kTtbqoDD4D54zY2-Rhu50WR5Nfdptq96K5JwLj7DxDM/edit

Reply:
Ok, just bug fixes.... and the rel notes themselves were merged yesterday yes?